styles/style.css

```diff
@@ -0,0 +1,80 @@
+h1 {
+  font-family: 'Noto Serif JP', sans-serif;
+}
+
+ul {
+  list-style-type: none;
+}
+
+nav ul li:hover {
+  text-decoration: underline;
+  color: white;
+}
+nav {
+  background-color: silver;
+}
+
+section {
```

Comment:
maybe add another selector for the "team-section" which makes border-bottom:none;